index.md

```diff
@@ -0,0 +1,178 @@
+
+Developers can use Facebook Ads to monetise their android apps using Facebook Audience Network. This encourages developers to create new applications on a regular basis.
+
+### Requirements analysis
+
+The key requirements for a successful integration include;
+    Facebook Audience Network SDK
+    Android studio
+    Facebook business account
+
+### Initial integration steps
+
+Before beginning the integration process, you have to first setup your Facebook Business Manager account for the ads.
+The steps below will guide you on setting up your account.
+
+####1. **Set-up a  Business Page**
+The first and fore-most step is to set up a business page where your ads will Appear. You can follow this short [video](https://www.youtube.com/watch?v=fzW4eHQQLFk) to see how you can create your own page.
+
+#### Step 2 **Set up a Facebook Manager  account**
+
+To create a Facebook manager account, go to https://business.facebook.com/overview to first create a Facebook manager account as shown in the picture below.
+![](engineering-education/integrating-facebook-ads-with-your-android-application-in-android-studio
+/create.png)
+
+#### Step 3 **Open a Facebook Audience Network (FAN) account**
+Set up a FAN account using the business manage account you created to make sure they are all integrated.
+
+
+
+#### Step 4 **Build Application properties for your account**
+
+Once you have created your FAN account, click on the extension/dropdown button at the top and select the monetization manager.
+Click on integration>properties, key in your Application name and id then select your platform, either Android or iOS.
+
+If you don't have a ready App, follow these [directives](#integration-in-android-studio) to create a simple test Application.
+
+#### Step 5 **Select the type of Ads**
+
+The final initial step is to select and create the ad type.
+Once done, you are ready for the integration process in the Android studio.
+
+### Requirements Installation
+
+Assuming you have your Android studio up and running, the only thing you have to install is the FAN SDK.
+
+ **FAN SDK installation**
+ To install, you can download your SDK manually from [here](https://developers.facebook.com/resources/audience-network-sdk-6.8.0.zip) (Not recommended) or follow the steps below to install it the right way.
+
+
+
+### Integration in Android Studio
+
+Don't have an Application? No need to worry since you can set-up an simple Application in a few steps below.
+
+#### Step 1: Create a New Project
+For those without a ready Application, open your Android Studio and create a new project with Ads activity. Since it is a test Application, you can use it without altering. This [article](https://www.section.io/engineering-education/first-android-App/) can be very useful in helping you with proper project creation.

```

Comment:
Please, you second person pronoun here "For those without a ready Application,". it should be "if you don't an application created already."